packages/state/recoil/selectors/wynd.ts

```diff
@@ -0,0 +1,104 @@
+import { selector, selectorFamily } from 'recoil'
+
+import {
+  AmountWithTimestampAndDenom,
+  WyndPools,
+  WyndPrices,
+} from '@dao-dao/types'
+import {
+  AssetInfo,
+  SwapOperation,
+} from '@dao-dao/types/contracts/WyndexMultiHop'
+import { WYND_API_BASE } from '@dao-dao/utils'
+
+import { refreshTokenUsdcPriceAtom } from '../atoms'
+
+export const wyndPoolsSelector = selector<WyndPools | undefined>({
+  key: 'wyndPools',
+  get: async () => {
+    try {
+      const wyndPools = await fetch(WYND_API_BASE + '/pools')
+      return (await wyndPools.json()) as WyndPools
+    } catch {
+      return undefined
+    }
+  },
+})
+
+export const wyndPricesSelector = selector<WyndPrices | undefined>({
+  key: 'wyndPrices',
+  get: async ({ get }) => {
+    get(refreshTokenUsdcPriceAtom(''))
+    try {
+      const wyndPools = await fetch(WYND_API_BASE + '/assets/prices')
+      return (await wyndPools.json()) as WyndPrices
+    } catch {
+      return undefined
+    }
+  },
+})
+
+// Price of 1 macro token of the specified denom or address.
+export const wyndUsdPriceSelector = selectorFamily<
+  AmountWithTimestampAndDenom | undefined,
+  // denomOrAddress
```

Comment:
what's this `denomOrAddress`? That's not immediately intuitive to me.

Reply:
denom refers to a native denom, such as `ujuno`, and address refers to a CW20 contract address. This terminology is used in many many places, and it also how WYND serves its prices (though it's just a string labeled `asset` in their API). `denomOrAddress` is commonly used throughout the UI so I think it's consistent 🤷